themes-default/slim/static/js/templates/app-header.vue

```diff
@@ -193,21 +206,6 @@ module.exports = {
     mounted() {
         const { $el } = this;
 
-        // Hover Dropdown for Nav
-        $($el).on({
-            mouseenter(event) {
-                const $target = $(event.currentTarget);
-                $target.find('.dropdown-menu').stop(true, true).delay(200).fadeIn(500, () => {
-                    $target.find('.dropdown-toggle').attr('aria-expanded', 'true');
```

Comment:
Setting `aria-expanded` on the `.dropdown-toggle` child to `'true'` is still needed.
You should be able to do that in `toggleDropdown` if you pass `$event` to it as a parameter.

Reply:
I'd say hooks are the way to go:
https://vuejs.org/v2/guide/transitions.html#JavaScript-Hooks